core/services/chainlink/config_general_test.go

```diff
@@ -30,6 +30,7 @@ func TestTOMLGeneralConfig_Defaults(t *testing.T) {
 	assert.False(t, config.StarkNetEnabled())
 	assert.Equal(t, false, config.JobPipeline().ExternalInitiatorsEnabled())
 	assert.Equal(t, 15*time.Minute, config.WebServer().SessionTimeout().Duration())
+	assert.Equal(t, false, config.HeadReport().TelemetryEnabled())
```

Comment:
@smartcontractkit/foundations do you think this should default to true? Or maybe a `HeadReport.TelemetryDisabled` config instead? I see this being enabled for every (prod) node by default

Reply:
I think the default should be `true`, as it requires less configuration to get this feature up and running